### DISCUSSION TITLE
图片预览改进, 可参考https://github.com/alx-xlx/goindex

### DISCUSSION THREAD

**Discussion by wimotek** [spencerwooo/onedrive-vercel-index#495] [Ideas]
图片预览改进, 可参考https://github.com/alx-xlx/goindex

**Comment by spencerwooo** [comment]
啥啊，就是那个之前我们用的 gallery 啊？有 Grid view 了不用那个了。

**Reply by LzNOHentai** [reply]
还是希望能够支持一下，因为想要获得近似看图APP的体验，方便网盘在线看漫画之类的。

**Reply by spencerwooo** [reply]
这不是本项目的功能重点，这样也只会让功能越来越复杂，因此不会做支持。
